src/lib/i18n/locales/fr-FR/translation.json

```diff
@@ -725,37 +725,37 @@
 	"Title": "Titre",
 	"Title (e.g. Tell me a fun fact)": "Titre (par ex. raconte-moi un fait amusant)",
-	"Title Auto-Generation": "Génération automatique de titres",
+	"Title Auto-Generation": "Génération automatique des titres",
 	"Title cannot be an empty string.": "Le titre ne peut pas être une chaîne de caractères vide.",
 	"Title Generation Prompt": "Prompt de génération de titre",
 	"To access the available model names for downloading,": "Pour accéder aux noms des modèles disponibles,",
 	"To access the GGUF models available for downloading,": "Pour accéder aux modèles GGUF disponibles,",
 	"To access the WebUI, please reach out to the administrator. Admins can manage user statuses from the Admin Panel.": "Pour accéder à l'interface Web, veuillez contacter l'administrateur. Les administrateurs peuvent gérer les statuts des utilisateurs depuis le panneau d'administration.",
 	"To attach knowledge base here, add them to the \"Knowledge\" workspace first.": "Pour attacher une base de connaissances ici, ajoutez-les d'abord à l'espace de travail « Connaissances ».",
 	"to chat input.": "Vers la zone de chat.",
-	"To protect your privacy, only ratings, model IDs, tags, and metadata are shared from your feedback—your chat logs remain private and are not included.": "",
+	"To protect your privacy, only ratings, model IDs, tags, and metadata are shared from your feedback—your chat logs remain private and are not included.": "Pour protéger votre confidentialité, seules les notes, les identifiants de modèle, les tags et les métadonnées de vos commentaires sont partagés. Vos journaux de discussion restent privés et ne sont pas inclus.",
 	"To select actions here, add them to the \"Functions\" workspace first.": "Pour sélectionner des actions ici, ajoutez-les d'abord à l'espace de travail « Fonctions ».",
 	"To select filters here, add them to the \"Functions\" workspace first.": "Pour sélectionner des filtres ici, ajoutez-les d'abord à l'espace de travail « Fonctions ». ",
-	"To select toolkits here, add them to the \"Tools\" workspace first.": "Pour sélectionner des toolkits ici, ajoutez-les d'abord à l'espace de travail « Outils ». ",
-	"Toast notifications for new updates": "",
+	"To select toolkits here, add them to the \"Tools\" workspace first.": "Pour sélectionner des outils ici, ajoutez-les d'abord à l'espace de travail « Outils ». ",
+	"Toast notifications for new updates": "Notifications toast pour les nouvelles mises à jour",
 	"Today": "Aujourd'hui",
-	"Toggle settings": "Basculer les paramètres",
-	"Toggle sidebar": "Basculer la barre latérale",
-	"Token": "",
-	"Tokens To Keep On Context Refresh (num_keep)": "Jeton à conserver lors du rafraîchissement du contexte (num_keep)",
-	"Too verbose": "",
-	"Tool": "",
+	"Toggle settings": "Afficher/masquer les paramètres",
+	"Toggle sidebar": "Afficher/masquer la barre latérale",
```

Comment:
"Afficher/masquer" (literally "Show/Hide") is much more used than the literal verb "Basculer"